python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -81,20 +82,22 @@ def manifest_path(self) -> Path:
         return self._manifest_path
 
     @public
-    def prepare(self, *, quiet=False) -> None:
+    def prepare_for_deployment(self, *, log_level=logging.INFO) -> None:
         """A method that can be called as part of the deployment process to handle
-        preparing the manifest and if packaged_project_dir is set, handle copying
-        the dbt project to that directory.
+        all potential preparation steps.
 
         Args:
-            quiet (bool):
-                Disable logging
-                Default: False
+            log_level (int):
+                Log level for messages.
+                Default: logging.INFO
         """
+        self.create_manifest(log_level)
+        self.sync_to_package_data(log_level)
+
+    @deprecated(breaking_version="1.7.0", additional_warn_text="use prepare_for_deployment")
```

Comment:
FWIW I would just change this and not worry about about backcompat, given that it was experimental and just there for a few releases